lib/libc/time/localtime.c

```diff
@@ -317,13 +299,9 @@ static void scrub_abbrs(struct state *sp) {
 
 static void update_tzname_etc(const struct state *sp,
                               const struct ttinfo *ttisp) {
-#if HAVE_TZNAME
```

Comment:
Wouldn't it be better (less changes) if you forced [HAVE_TZNAME](https://mimiker.ii.uni.wroc.pl/source/xref/mimiker/lib/libc/time/private.h?r=10da8877#208) and [USG_COMPAT](https://mimiker.ii.uni.wroc.pl/source/xref/mimiker/lib/libc/time/private.h?r=10da8877#200) to be defined?

Reply:
I've done it in such a way in https://github.com/cahirwpz/mimiker/pull/1361/commits/1b55b6fab21ebe4e6a9c112583dc0cd1014b0499.